be/src/exprs/bitmap_function.cpp

```diff
@@ -343,14 +367,19 @@ StringVal BitmapFunctions::bitmap_hash(doris_udf::FunctionContext* ctx, const do
 }
 
 StringVal BitmapFunctions::bitmap_serialize(FunctionContext* ctx, const StringVal& src) {
-    auto src_bitmap = reinterpret_cast<BitmapValue*>(src.ptr);
-    StringVal result = serialize(ctx, src_bitmap);
-    delete src_bitmap;
-    return result;
+    if (src.is_null) {
+        BitmapValue src_bitmap;
+        return serialize(ctx, &src_bitmap);
+    } else {
+        auto src_bitmap = reinterpret_cast<BitmapValue*>(src.ptr);
+        StringVal result = serialize(ctx, src_bitmap);
+        delete src_bitmap;
```

Comment:
why delete the src_bitmap here? who will manage the bitmap lifetime?

Reply:
The src_bitmap was initialized in merge function. So it need to delete src_bitmap when UDA has been finished.